packages/ag-charts-community/src/chart/toolbar/toolbarStyles.css

```diff
@@ -144,9 +144,9 @@
    differs per theme. */
 .ag-charts-toolbar__button--with-transition {
     transition:
-        background-color 0.25s ease-in-out,
-        border-color 0.25s ease-in-out,
-        color 0.25s ease-in-out;
+        background-color 1s ease-in-out,
+        border-color 1s ease-in-out,
+        color 1s ease-in-out;
```

Comment:
Should these be changed? (Guessing this was for debugging?)

Reply:
Good catch